package com.demo.exception;

import java.io.FileNotFoundException;

public class ExceptionHandlingDemo {

	public static void main(String[] args)  {
		// TODO Auto-generated method stub
		try {
			
			
		BackendApplication.send(1, "", "http://abc.test.com");
		} catch (FileNotFoundException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
	System.out.println("In main : after exception");
	}

}
